oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/LogoutHandler.java

```diff
@@ -120,7 +120,7 @@ public APIGatewayProxyResponseEvent logoutRequestHandler(APIGatewayProxyRequestE
     }
 
     private APIGatewayProxyResponseEvent processLogoutRequest(
-            Session session, APIGatewayProxyRequestEvent input, Optional<String> state) {
+            Session session, APIGatewayProxyRequestEvent input, Optional<String> state) throws ParseException {
```

Comment:
What happens if this throws a ParseException?

Reply:
Parse exception was from validating id_token_hint . Removed the catch exception as it should be caught in the validation function. But have added it back in as we dont explicitly check the parse error for when we grab the aud from the JWTclaims.

Reply:
So we get an `error logout response` for unable to validate which includes the parse exception for the idToken. Have added back in a try catch for the parse error on the `aud` so we will get an `error logout response`

Reply:
This "throws ParseException" will be removed as it is now caught